data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -91,6 +91,32 @@
     "Low median household income as a percent of area median income"
 )
 
+# Additional ACS demographic fields.
+BLACK_FIELD_NAME = "Black or African Amer."
```

Comment:
See notes above but also ty for pulling this out to field names!

We also might not need the population count fields, and can keep just the percentage fields. Wdyt? The total_population + percentages should be comprehensive.

Reply:
Oooh good point. I'll change that.